update-scripts/rust-analyzer/default.nix

```diff
@@ -124,6 +134,18 @@ let
       type ? null,
     }:
     let
+      filteredMarkdownDesc =
+        # If there is a risk that the string contains an heading filter it out
```

Comment:
Can't we just do a line-wize regex replace converting headings to bold lines? Pandoc feels like overkill here just to filter out lines starting with `#`

Reply:
Essentially `sed -e 's/^#\+ \(.*\)/**\1**/`

Reply:
We should be able to do a replacement like that in nix, without resorting to another IFD

Reply:
Well yeah, but that would risk changing stuff in code blocks

Reply:
Ah I didn't think of code blocks! This is fine for now then I guess.

I'll look in a bit more detail once I get back

Reply:
We could make an option in pure nix, but it would be a more complicated implementation.

I could look into it, maybe it would not be as complicated as I think

Reply:
TBH I don't think it's worth spending time on. This is only really built on CI which doesn't seem to be struggling currently.